zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -215,27 +216,30 @@ def channelmessage2zerver_message(slack_dir, channel, added_users, added_channel
     msg_id_count = 1
 
     # Sanitize the message text
-    def sanitize(text):
-        saved_string = str(text)
+    def sanitize_token(text):
         text = text.split(' ')
-        text[0] = text[0].replace('<@', ' ')
-        text[0] = text[0].replace('>', ' ')
-        text[0] = text[0].replace('|', ' ')
-        length = len(text[0].split(' '))
-        if length > 1:
-            try:
-                short_name = text[0].split(' ')[2]
-            except IndexError:
-                short_name = ''
-            text[0] = text[0].split(' ')[1]
-        for user in users:
-            if (user['id'] == text[0] and user['name'] == short_name and length == 4) or \
-               (user['id'] == text[0] and length == 3):
-                text[0] = user.get('real_name', user['name'])
-                text[0] = "@**" + text[0] + "** "
-                text = " ".join(text)
-                return text
-        return saved_string
+        token_index = 0
+        for token in text:
+            if (re.compile(r"<@.*|.*>").match(token)):
```

Comment:
I meant, L223 until L239 becomes sanitize_token(token) which returns the sanitized token.

Reply:
The code should be structured as:
```
tokens = text.split(' ')
text = ' '.join([sanitize_token(t) for t in tokens])
```